apps/frontpage/next.config.js

```diff
@@ -48,4 +39,345 @@ module.exports = {
+        source: '/docs/react/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/vue/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/angular/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/web-components/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/ember/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/html/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/mithril/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/marko/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/svelte/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/riot/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/preact/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      {
+        source: '/docs/rax/get-started/examples',
+        destination: '/showcase',
+        permanent: true,
+      },
+      ////// Addons ///////////////////////////////////
+      {
+        source: '/addons/addon-gallery',
+        destination: '/integrations',
+        permanent: true,
+      },
+      {
+        source: '/addons',
+        destination: '/integrations',
+        permanent: true,
+      },
+      {
+        source: '/recipes',
+        destination: '/integrations',
+        permanent: true,
+      },
+      {
+        source: '/addons/tags/:tag',
+        destination: '/integrations/tag/:tag',
+        permanent: true,
+      },
+      /////////////////////////////////////////////////
+      /////////////// 🐺 Wild Cards ///////////////////
+      /////////////////////////////////////////////////
+      {
+        source: '/basics/:path*',
+        destination: '/docs',
+        permanent: true,
+      },
+      {
+        source: '/docs/basics/:path*',
+        destination: '/docs',
+        permanent: true,
+      },
+      {
+        source: '/configurations/:path*',
+        destination: '/docs/configure',
+        permanent: true,
+      },
+      {
+        source: '/examples/:path*',
+        destination: '/docs',
+        permanent: true,
+      },
+      {
+        source: '/logos/:path*',
+        destination: '/docs',
+        permanent: true,
+      },
+      {
+        source: '/testing/:path*',
+        destination: '/docs',
+        permanent: true,
+      },
```

Comment:
These all need the `/docs/...` versions, too:

```suggestion
      {
        source: '/configurations/:path*',
        destination: '/docs/configure',
        permanent: true,
      },
      {
        source: '/docs/configurations/:path*',
        destination: '/docs/configure',
        permanent: true,
      },
      {
        source: '/examples/:path*',
        destination: '/docs',
        permanent: true,
      },
      {
        source: '/docs/examples/:path*',
        destination: '/docs',
        permanent: true,
      },
      {
        source: '/logos/:path*',
        destination: '/docs',
        permanent: true,
      },
      {
        source: '/docs/logos/:path*',
        destination: '/docs',
        permanent: true,
      },
      {
        source: '/testing/:path*',
        destination: '/docs',
        permanent: true,
      },
      {
        source: '/docs/testing/:path*',
        destination: '/docs/writing-tests',
        permanent: true,
      },
```